agent/softwarecontainer-config.in

```diff
@@ -1,27 +1,35 @@
+
+#
+# General SoftwareContainer configuration.
+#
+# Optional values are commented out. All other values are mandatory.
```

Comment:
Like: "Optional configs are commented out, but are set to the default values that will be used by SoftwareContainer if nothing else is specified." ?